### DISCUSSION TITLE
Does Dialogic support Godot 4.1?

### DISCUSSION THREAD

**Discussion by Theooolone** [dialogic-godot/dialogic#1708] [Q&A] [answered]
I'm developing a small game in Godot 4.1 and Dialogic would be perfect for it, does Dialogic 2.0 support Godot 4.1?

If not, what do the creators of Dialogic have planned for Godot 4.1?

**Comment by Jowan-Spooner** [comment, marked as answer]
Yes the latest alpha9 release works on 4.1.

**Reply by Theooolone** [reply]
This is fantastic, thank you for the help!
